src/flow/ext_multiroute_flow.jl

```diff
@@ -181,7 +181,7 @@ function slope{T<:AbstractFloat}(
   )
   slope = 0
   for e in edges(flow_graph)
-    if cut[dst(e)] - cut[src(e)] > 0 &&
+    if cut[dst(e)] == 2 > cut[src(e)] &&
```

Comment:
Well, we can chain comparison in Julia, so it seemed OK to me.
`cut[dst(e)] == 2 > cut[src(e)]` is the same as `cut[dst(e)] == 2 && 2 > cut[src(e)]`, isn't it?

Anyway, the idea is, if the head of edge e in on the same side of the cut as the source and the destination of e in the same side of the cut as the target, then if the capacity of this edge is strictly bigger than the restriction value, then I increment the value slope.